action.sh

```
@@ -0,0 +1,8 @@
+#!/system/bin/sh
+MODDIR=/data/adb/modules/SMTW
+function abort() {
+    echo "- $1"
+    sleep 0.5
+    exit 1
+}
+sh "$MODDIR/SuperMario-Tweaker.sh" &>/dev/null || abort "- Failed to apply tweaks"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)